### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by faho** [kwin-scripts/kwin-tiling#21]
As we've learned in #18 and in various comments, this script can be fairly confusing, especially for users new to tiling.

I propose (and will probably also implement, but anyone reading this feel free to help) to create more/better documentation and to do it in one place. Since I don't have any actual site of my own, I'll probably do it here on github, and the README.org is a good start.

So, what needs to be documented?
- An introduction to tiling
- The various layouts
- The resizing functionality
- The various hotkeys
- The concept of the "master" window

**Comment by timlepes** [comment]
Yes, documentation would help.  I have not been a tiling WM user in the past, though I have become interested in tiling.  I have looked at i3 and Sway, but don't really want to switch away from Plasma.  I haven't needed a cheat-sheet printed out since the 80's... but i3/Sway take getting used to.  Adding tiling features to KWin with this script sounds like exactly what I am looking for.  But how to make it work?  Even some simple docs, or a link to a YouTube or Vimeo video giving an overveiew would be very helpful and appreciated.  Consider this my Plus-1 or up-vote.  Thanks!

**Comment by bitsofreedom** [comment]
You should also mention that users should enable the 'Tiling Extension'. It was not enabled by default for me.

**Comment by jflorian** [comment]
Also explicitly mention when to expect it to become effective.  Specifically, would this be once the install is complete or is a restart of plasma required?

**Comment by laloch** [comment]
@jflorian, I have updated the README (see 48a564f50fc9869b86e2aba99ceb1ef39a206fe0). The KCM dialog is self-explanatory in my opinion, so no need to further instruct the user.
